BLUESPAWN-common/headers/common/wrappers.hpp

```diff
@@ -84,6 +84,14 @@ class FindWrapper : public GenericWrapper<HANDLE> {
 		GenericWrapper(handle, std::function<void(HANDLE)>(FindClose), INVALID_HANDLE_VALUE){};
 };
 
+class AcquireMutex : public GenericWrapper<HANDLE> {
+public:
+	AcquireMutex(HANDLE hMutex) :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'FindWrapper' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5011005)